setup.cfg

```diff
@@ -23,4 +23,11 @@ classifiers =
     Topic :: Scientific/Engineering
 project_urls =
     Release notes = https://github.com/explosion/weasel/releases
-    Source = https://github.com/explosion/weasel/
\ No newline at end of file
+    Source = https://github.com/explosion/weasel/
+
+[options.entry_points]
+console_scripts =
+    weasel = weasel._util:app
+    # TODO: Using 'spacy' as an alias has consequences,
+    # and may not be the best approache
+    spacy = weasel._util:app
```

Comment:
Is there a way to reverse the logic and have spaCy deal with this instead? Ideally, I'd want `weasel` to be ignorant of spaCy, if you know what I mean.

Reply:
Absolutely, it makes sense. As discussed IRL, we can do something like:

```python
from weasel.cli import app as project_cli

app.add_typer(project_cli, name="project")
```

on the spaCy side. That way `python -m spacy project` will seamlessly run Weasel under the hood.

I'll change this PR accordingly.